### PR TITLE
Add onboarding preparation docs

### PR DESCRIPTION
**Pull request by dadlerj**

### CODE REVIEW

File: handbook/people-ops/onboarding/before_starting.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+### Immediately after offer signing
+
+- Each new teammate is added to [teammate tracking spreadsheet](https://docs.google.com/spreadsheets/d/1qt3rGNRVaSyi_N2uN3DaXvlg8LjfKLJb8PFfrQsmR-I/edit#gid=948830545) (name, title, start date, location)
+
+### One week prior to first day
+
+- Each new teammate receives a welcome email a week before their start date providing information and asking questions about benefits, email address selection, equipment purchases, and other onboarding logistics (we provide two templates, for US and non-US teammates)
+  - [US template](https://docs.google.com/document/d/1a3uhkt0bvmMuTmFBvroEXSdQGIPwCz8BLgk_YiSMZUQ/edit)
+  - [Non-US template](https://docs.google.com/document/d/1OxrGknE6NKoeMY7qMDe5th8ZL3juxFOKfH-eywWuwko/edit)
+  - _As of 2021-01-18, all hiring managers are CCed on these emails_
```
Comment (christinaforney) [inline handbook/people-ops/onboarding/before_starting.md:22]: 🎉

**Review by christinaforney — APPROVED** [review]
Thank you for doing this!

Buddy assignment seems to be missing from this list. Is PeopleOps responsible for scheduling the recurring meetings? Who is responsible for communicating/confirming who the buddy is to the new hire?

**Comment by dadlerj** [comment]
Good call, will add notes on timing For selecting a buddy. For any info on how it works, this page has some details: https://about.sourcegraph.com/handbook/people-ops/onboarding/buddy-program

File: handbook/people-ops/onboarding/before_starting.md (hunk 1 above)
Comment (mercadon) [reply handbook/people-ops/onboarding/before_starting.md:22]: I was making a couple of edits to our buddy page but those are done now and I'll add in that section here too!

**Review by mercadon — APPROVED** [review]

File: handbook/people-ops/onboarding/before_starting.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+- Each new teammate is added to a short list of general onboarding accounts
+  - Gmail + any specific mailing groups
+    - _As of 2021-01-15, all hiring managers will be notified of the email addresses going live upon creation_
+  - Slack + any specific channels and/or teams
+  - GitHub (if they already have an account)
+  - 1Password
+  - Zoom
+  - Lever
+  - Lattice
+- New hires are assigned an onboarding buddy by their manager (People Ops will reach out to managers via Slack to confirm onboarding buddy)
```
Comment (dadlerj) [inline handbook/people-ops/onboarding/before_starting.md:38]:
```suggestion
- Each new teammate is assigned an onboarding buddy by their manager (People Ops will reach out to managers via Slack to confirm the onboarding buddy)
```

**Pull request merged**